public/app/features/dashboard-scene/scene/DashboardScene.tsx

```diff
@@ -166,6 +170,8 @@ export class DashboardScene extends SceneObjectBase<DashboardSceneState> {
 
     if (this.state.isEditing) {
       this._changeTracker.startTrackingChanges();
+
+      this.state.scopes?.enterViewMode();
```

Comment:
Think would be nicce if we reverSed the dependency here, and rather than controling this from the DashbaordScene, inseat subscribe to the dashboard scene in the SceneScopes and enterViewMode from there. This way the changes are going to be a bit better encapsulated.

Reply:
Yup. I'll fix this pretty fast.